includes/class-image-optimisation.php

```
@@ -95,65 +95,112 @@ public function maybe_serve_next_gen_images( $buffer ) {
+								$original_token = $parts[0];
+								$normalized_url = $this->normalize_url( $original_token );
+								$descriptor     = $parts[1];
+
+								if ( $this->is_valid_url( $normalized_url ) ) {
+									$new_url = $this->replace_image_with_next_gen( $normalized_url, $exclude_imgs, $supports_avif, $supports_webp );
+									// Use the optimized URL if conversion happened, otherwise keep the original token.
+									$final_url          = ( $new_url !== $normalized_url ) ? $new_url : $original_token;
+									$new_srcset_parts[] = $final_url . ( $descriptor ? " $descriptor" : '' );
+								} else {
+									$new_srcset_parts[] = $original_token . ( $descriptor ? " $descriptor" : '' );
+								}
```

PHPMD (2.15.0)

[error] The method maybe_serve_next_gen_images uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)